main/FAB/index.tsx

```diff
@@ -0,0 +1,127 @@
+import {Animated, Easing, StyleProp, View, ViewStyle} from 'react-native';
+import {ButtonSize, IconButton} from '../IconButton';
+import {DoobooTheme, withTheme} from '../theme';
+import {Icon, IconName} from '../Icon';
+import React, {ReactElement, useLayoutEffect, useMemo, useRef} from 'react';
+
+import styled from '@emotion/native';
+
+export const StyledIcon = styled(Icon)`
+  color: ${({theme}) => theme.textContrast};
+`;
+export interface FABItem {
+  icon: IconName;
+  id: string;
+}
+
+export interface FABProps<Item extends FABItem> {
+  isActive: boolean;
+  fabItems: Item[];
+  onPressFAB: () => void;
+  onPressFabItem: (item?: Item) => void;
+  renderFAB?: () => ReactElement;
+  renderFabItem?: (item: Item, idx: number) => ReactElement;
+  size: ButtonSize;
+  style?: StyleProp<ViewStyle>;
+}
+
+function FloatingActionButtons<Item extends FABItem = FABItem>({
+  isActive,
+  fabItems,
+  onPressFAB,
+  onPressFabItem,
+  renderFAB,
+  renderFabItem,
+  size = 'large',
```

Comment:
Although they might have same size in general, I think `FABItem` is responsible for setting size, not `FloatingActionButtons` component.